modules/scheduling/kube-green/base/README.md

```diff
@@ -0,0 +1,28 @@
+# Traefik Module
+
+Kube-green is a Kubernetes addon that allows you to shut down the workload you don't need with the definition of *schedules*.
+
+[Here is the link to Kube-green official repository.]
+
+## What the base module contains
+
+- **CRDs:** Kube-green custom resource definitions
+- **Resources (Traefik ingress):**
+  - **Configs:** configurations of related resources.
```

Comment:
Better be explicit that we deploy a validation webhook for the kube-green resources